app/components/Mods/ModsPage.tsx

```diff
@@ -36,11 +43,14 @@ const ModsPage: React.FunctionComponent = () => {
           highlighted
         />
         {isEmpty && (
-          <Card>
-            <Typography variant="h6" align="center">
-              {modsText.emptyModList}
-            </Typography>
-          </Card>
+          <div>
+            <br/>
```

Comment:
I didn't check exactly why you needed this, but doesn't look good. Don't use `br` to add spacing, `br` has a semantic meaning. Either add a top margin to the Card component, or add a Box wrapper with that margin.